### DISCUSSION TITLE
Password stored in clear text?

### DISCUSSION THREAD

**Discussion by perfecto25** [rustdesk/rustdesk#780] [General]
running RD 1.1.9 on ubuntu 22, was looking at config toml,
```
cat /root/.config/rustdesk/RustDesk.toml 
id = '295513710'
password = 'MyPassword123'
salt = 'mktuzy'
key_pair = [
    [
    30,
    160,
    30,
    18,
```

the password is stored as clear text, is this intentional?

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/721#discussioncomment-2922061

**Comment by rustdesk** [comment]
https://github.com/users/rustdesk/projects/15
